word-search-page/word-search.js

```diff
@@ -0,0 +1,36 @@
+//Selectors
+function get(id1) {
+    return document.getElementById(id1);
+}
+let inputValue = get("search-input");
+let searchButton = get("search-button");
+let searchResult = get("search-result");
+let addResult = document.createElement('li');
+let word = inputValue.value;
```

Comment:
Don't get the input value here because it is empty.
 you have to get the input value after it is typed so, get it when the click action happens but firstly handle the case if the search-button is clicked without typing a value by notifying the user to enter a valid value without  making a useless request to fetch data.